op-mode-definitions/tinc.xml.in

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="interfaces">
+        <children>
+          <node name="tinc">
+            <properties>
+              <help>Show Tinc VPN interface information</help>
```

Comment:
Another minor point: here it's sometimes "Tinc" and sometimes "tinc". It's better to make it consistent. On https://tinc-vpn.org/ the official styling seems to be all-lowercase "tinc", they even use it when "tinc" is the first word of a sentence and break the convention that sentences must start with a capital letter to maintain that styling.

I believe we should be using all-lowercase "tinc" everywhere.

Reply:
done